### PR TITLE
Release of 2.0.0 Stable

### PR DESCRIPTION
**Pull request by myungjoo**
This release is being drafted.

Release Target
- Date: 2021-09 (accompanied by announcement of LF and OSG) // Confirmed by LF/OSG: 2021-10-01
- Along with Tizen 6.5 M2 release (2.0.y)

TODO
- [x] check if all required 2.0.0 features are merged.
- [x] write CHANGES of 1.6.y --> 2.0.y, not 1.7.2 --> 2.0.0 and release notes and other docs.
- [x] notify LF and OSG. (sent 8/30. waiting for replies)
- [x] wiki write-ups

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,32 @@
+1.7.2 -> 2.0.0
+	- NNStreamer for Edge-AI
+		- MQTT Pub/Sub streams can be synchronized with timestamp values
+		- MQTT-Hybrid Pub/Sub protocol (send high-bandwidth data streams with HLS) is not included in 2.0. This feature will be enabled with 2.1+ releases.
+		- Query (workload offloading) with TCP is included as an experimental feature. Query will be updated to use MQTT-Hybrid in future 2.1+ releases.
+
+	- Stream data type redefined (backward compatible)
+		- Flex-tensor and Sparse-tensor as a format of "other/tensors".
+		- "other/tensor" is obsoleted (but will be kept supported for a while). Use "other/tensros,num_tensors=1" for single tensor stream.
```
Comment (jaeyun-jung) [inline CHANGES:9]: Use "other/tensros : typo tensros

Comment (myungjoo) [reply CHANGES]: Fixed. Thanks.

**Review by anyj0527 — APPROVED** [review]
COOL 👍

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**